KsApi/models/Config.swift

```diff
@@ -33,6 +33,44 @@ public struct Config {
   }
 }
 
+extension Config: Swift.Decodable {
+
+  enum CodingKeys: String, CodingKey {
+    case abExperiments = "ab_experiments"
+    case appId = "app_id"
+    case applePayCountries = "apple_pay_countries"
+    case countryCode = "country_code"
+    case features
+    case iTunesLink = "itunes_link"
+    case launchedCountries = "launched_countries"
+    case locale
+    case stripe
+
+    enum StripeCodingKeys: String, CodingKey {
+      case stripePublishableKey = "publishable_key"
+    }
+  }
+
+  public init(from decoder: Decoder) throws {
+    let values = try decoder.container(keyedBy: CodingKeys.self)
+    self.abExperiments = try values.decode([String: String].self, forKey: .abExperiments)
+    self.appId = try values.decode(Int.self, forKey: .appId)
+    self.applePayCountries = try values.decode([String].self, forKey: .applePayCountries)
+    self.countryCode = try values.decode(String.self, forKey: .countryCode)
+    self.features = try values.decode([String: Bool].self, forKey: .features)
+    self.iTunesLink = try values.decode(String.self, forKey: .iTunesLink)
+    self.launchedCountries = try values.decode([Project.Country].self, forKey: .launchedCountries)
+    self.locale = try values.decode(String.self, forKey: .locale)
+
+    let stripe = try values.decode([String: String].self, forKey: .stripe)
```

Comment:
I think we can replace the `if/else` here and set the property directly with:
```swift
self.stripePublishableKey = try container
  .nestedContainer(keyedBy: StripeCodingKeys.self, forKey: .stripe)
  .decode(String.self, forKey: .stripePublishableKey)
```

Reply:
Nice solution!